bundles/org.openhab.binding.caddx/src/main/resources/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="caddx" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0" 
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+		<label>Caddx</label>
+		<description>This bridge represents the Caddx Serial interface.</description>
+
+		<channels>
+			<channel id="bridge_reset" typeId="reset">
+				<label>Reset Bridge</label>
+				<description>Resets the communication with the panel</description>
+			</channel>
+			<channel id="send_command" typeId="command">
+				<label>Send Command</label>
+				<description>Sends an Alarm Command</description>
+			</channel>
+		</channels>
+
+		<config-description>
+			<parameter name="serialPort" type="text" required="true">
+				<context>serial-port</context>
+				<label>Caddx Bridge Serial Port</label>
+				<description>The serial port name for the communication. Valid values
+					are e.g. COM1 for Windows and /dev/ttyS0 or /dev/ttyUSB0 for Linux.</description>
+			</parameter>
+
+			<parameter name="baudrate" type="integer" required="false">
+				<label>Baud Rate</label>
+				<description>The baud rate of the serial port. Valid values are 9600
+					(default), 19200, 38400, 57600, and 115200.</description>
+				<default>9600</default>
+			</parameter>
```

Comment:
You should add options here so the user can select the values from a dropdown.
```suggestion
				<default>9600</default>
				<options>
                                 ...
				</options>
			</parameter>
```

Reply:
Done